code/game/area/Space_Station_13_areas.dm

```diff
@@ -1025,6 +1025,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "Brig"
 	icon_state = "brig"
 	mood_bonus = -3
+	mood_job_allowed = list(JOB_NAME_HEADOFSECURITY,JOB_NAME_WARDEN)
```

Comment:
this still needs detective, secoff, and brigphys. brig is a massive area including the outside area